Tools/MiniBrowser/wpe/main.cpp

```diff
@@ -514,6 +521,34 @@ static void activate(GApplication* application, WPEToolingBackends::ViewBackend*
 #endif
 }
 
+static gboolean parseWindowSize()
+{
+    if (!windowSize)
+        return FALSE;
+
+    gchar** windowSizes = g_strsplit(windowSize, "x", 2);
```

Comment:
`windowSizes` needs to be freed with `g_strfreev()`

Reply:
Oh good catch, I used `GUniquePtr<const char*>` before which is not available in `Tools/MiniBrowser`. Will fix.

Reply:
Will switch to `g_auto(GStrv) = ...` as suggested by @GeorgesStavracas :-)